contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java

```diff
@@ -67,9 +68,17 @@
 /** HTML page formatter for all exported {@link View}s. */
 final class StatszZPageHandler extends ZPageHandler {
 
+  private static final Object monitor = new Object();
+
   private final ViewManager viewManager;
+
+  @GuardedBy("monitor")
   private final Map<String, Measure> measures = Maps.newTreeMap();
+
+  @GuardedBy("monitor")
   private final Set<View> cachedViews = Sets.newHashSet();
+
+  @GuardedBy("monitor")
   private final TreeNode root = new TreeNode();
```

Comment:
It would help to have a comment explaining the purpose of `measures`, `cachedViews`, and `root` and how they are updated, though it doesn't need to be in this PR.

Reply:
Sure, I added some comments regarding that.